linux_os/guide/services/ssh/ssh_server/sshd_use_strong_ciphers/oval/shared.xml

```diff
@@ -1 +1 @@
-{{{ oval_sshd_config(parameter="Ciphers", value="((chacha20-poly1305@openssh\.com|aes256-gcm@openssh\.com|aes128-gcm@openssh\.com|aes256-ctr|aes128-ctr),?)") }}}
+{{{ oval_sshd_config(parameter="Ciphers", value="((aes128-ctr|aes192-ctr|aes256-ctr|chacha20-poly1305@openssh\.com|aes256-gcm@openssh\.com|aes128-gcm@openssh\.com|aes256-ctr|aes128-ctr),?)") }}}
```

Comment:
aes128-ctr and aes256-ctr is twice in the list

Reply:
the rule.yml also has it twice and the OCIL there is missing aes192-ctr, could you please fix rule.yml as well?

Reply:
And the OVAL has been missing + as well. I will indeed add some scenarios :-)